10.md

```diff
@@ -53,6 +53,34 @@ When replying to a text event E the reply event's "p" tags should contain all of
 Example:  Given a text event authored by `a1` with "p" tags [`p1`, `p2`, `p3`] then the "p" tags of the reply should be [`a1`, `p1`, `p2`, `p3`]
 in no particular order.
 
+## Plain text
+
+"Plain text" content is distinguished from content that requires parsing in order to be human-readable. Plain text MAY have some light formatting (as defined below), as long as it can be naturally read by humans as written. Client MAY parse and display notes to improve the aesthetic of displayed content.
+
+The litmus test for formatting _implemented by clients_ is whether such syntax would naturally be used by non-technical users with no expectation of parsing or rendering. Users themselves MAY include any formatting they choose, but formatting implemented by clients MUST be limited to the following:
+
+- A `>` at the beginning of a line MAY be used to indicate quotations
+- A `#` at the beginning of a word MAY be used to indicate a topic and SHOULD have a corresponding `t` tag
+- A `$` at the beginning of a word MAY be used to indicate a stock ticker
+- Matched `*` characters within a single line MAY be used to indicate emphasis
+- Matched `:` characters around a single word MAY be used to indicate emojis (see [NIP 30](./30.md))
+- Single-backticks with no language tag MAY be used for inline code blocks
+- Triple-backticks with no language tag MAY be used for multi-line code blocks
+- Unordered lists MUST be one level deep, and SHOULD use the `-` character
+- Ordered lists MUST be one level deep, and MUST be sequentially numbered
+
+Note that code blocks are only admissible because they are useful for opting-out of any default parsing clients may do.
+
+The biggest departure of this spec from markdown is the omission of special syntax for links. The reason for this is that markdown's link syntax is not idiomatic - in other words, humans do not generally use markdown's link syntax to denote a link. However, links are an essential part of internet content, and so for the purposes of this spec, URLs less than 300 characters may be considered "human-readable". Note that this specifically excludes embedded base64 image data and `nembed`s.
+
+Examples of links:
+
+- [NIP 73](./73.md) identifiers
+- Nostr entities with or without a `nostr:` prefix
```

Comment:
```suggestion
- Nostr entities with a `nostr:` prefix
```

Reply:
If the justification is that people are going to paste stuff, priming clients to expect nostr entities sans prefix is probably a good idea. Same way we approach lnurl etc

Reply:
Ok, I just checked and these almost universally have the prefix.

Reply:
many clients add the prefix when the user clicks to copy, many clients do not render the embedded mention when there is no prefix, some others that are currently committing the crime of rendering embeds without prefix have pledged to start behaving better this week after I asked.

there are also clients that force-prepend the "nostr:" prefix whenever a naked nip19 code is pasted, which I also think is a good approach because it keeps everything saner for other clients. maybe they should also force-prepend a "https://" whenever a domain is pasted without the scheme (or refuse to render domain as URLs). this is what Twitter used to do.